### PR TITLE
feat: pod home

### PR DESCRIPTION
**Pull request by juniusfree**
## :pushpin: References

- **Wonder issue:** https://app.wonderverse.xyz/dashboard?task=77029773482656107
- **Related pull-requests:** https://github.com/wondrous-dev/wondrous-backend/pull/675

## :blue_book: Description

Add the profile/home page to pods

## :movie_camera: Screenshot or Video

image

## :cake: Checklist:

- [x] I self-reviewed my changes
- [x] My changes follow the style guide: https://creative-earth-33e.notion.site/TT-Wonder-Style-guide-4702a45133374148953bfcaf584120b7
- [x] I tested my changes in Chrome

### CODE REVIEW

File: wondrous-app/components/Common/SidebarEntityList/index.tsx [hunk 1]
```diff
@@ -73,7 +73,7 @@ const useSidebarData = () => {
     ONLY_GRANTS_ENABLED_ORGS.includes(board?.orgData?.id) || ONLY_GRANTS_ENABLED_ORGS.includes(board?.orgId);
 
   const data = [
-    !!(orgBoard && !board?.orgData?.shared) && {
```
Comment (terryli0095) [inline wondrous-app/components/Common/SidebarEntityList/index.tsx:76]: i think we need this

File: wondrous-app/components/ProjectProfile/types.ts [hunk 2]
```diff
@@ -2,11 +2,13 @@ import { NextRouter } from 'next/router';
...
 import { ENTITIES_TYPES } from 'utils/constants';
 
+export type EntitiesType = typeof ENTITIES_TYPES[keyof typeof ENTITIES_TYPES] | null;
+
 export interface IListItemProps {
   LeftComponent: React.ElementType;
   RightComponent?: React.ElementType;
   data?: object;
-  onClick?: (router: NextRouter, data?) => void;
+  onClick?: (params: { router?: NextRouter; data?; entityLink?: string }) => unknown;
```
Comment (terryli0095) [inline wondrous-app/components/ProjectProfile/types.ts:11]: im not too familiar with this why do we use unknown vs any?

**Comment by terryli0095** [comment]
this breaks the sidebar navigation, since we use orgData and not projectData

File: wondrous-app/components/Common/SidebarEntityList/index.tsx (hunk 1 above)
Comment (juniusfree) [reply wondrous-app/components/Common/SidebarEntityList/index.tsx:76]: Are we going to show this when in pods?

File: wondrous-app/components/ProjectProfile/types.ts (hunk 2 above)
Comment (juniusfree) [reply wondrous-app/components/ProjectProfile/types.ts]: actually, I changed this to `void`.

`unknown` is like `any` but you can't do anything with it. https://www.typescriptlang.org/docs/handbook/2/functions.html#unknown

**Pull request merged**